app/models/lunch.rb

```diff
@@ -106,5 +120,13 @@ def tag_food(food)
         { title: I18n.t(title), summary: food, price: 85 }
       end
 
+      def get_allergens(meal_entry)
```

Comment:
Apply a bit of functional programming here!

```ruby
ALLERGENS_IMAGES.select do |image_name, v| 
  meal_entry[:summary].include? image_name
end.map { |k, allergen| allergen }
```

And just return the array instead!

Move the formatting: comma separation and prepend with `-` in the view instead 👍